python/cudf/cudf/utils/dtypes.py

```diff
@@ -16,6 +16,7 @@
 from cudf.core.missing import NA
 
 _NA_REP = "<NA>"
+_NAT_REP = "NaT"
```

Comment:
I'm not _really_ convinced that we need `_NA_REP`. You can write `str(cudf.NA)` to get this value. Same for NaT. Is this actually helpful, or does it just result in needless imports when you can call `str`? (This can be refactored in a separate PR if you agree it's a good idea.)